### PR TITLE
Added instructions to run Forseti from a non-org root using Terraform

### PR DESCRIPTION
**Pull request by red2k18**
Added instructions to run Forseti from a non-org root using Terraform

### CODE REVIEW

File: _docs/_latest/configure/general/non-org-root.md [hunk 1]
```diff
@@ -28,57 +28,38 @@ manually assign the correct roles later.
 
 ## Configure Forseti to Run on a Folder
 
-1. Edit `forseti_conf_server.yaml` and point the `root_resource_id`
-to the target folder:
-`folders/<foo_folder_id>`.
+1. Edit `main.tf` and set `composite_root_resources` variable to the target
```
Comment (blueandgold) [inline _docs/_latest/configure/general/non-org-root.md]: nit: `...point to the target folder`

File: _docs/_latest/configure/general/non-org-root.md [hunk 2]
```diff
@@ -28,57 +28,38 @@ manually assign the correct roles later.
...
+include one or more resources from GCP resource hierarchy in a single Forseti 
+installation.
 See [Configure Inventory]({% link _docs/latest/configure/inventory/index.md %})
 for more details.
 
-1. This assumes that Forseti is not installed with Org Admin credential, and
-you want Forseti to run on projects that you own. If Forseti is installed
-with Org Admin credential, then all the resources in the organization
-will be returned.
+For example: `composite_root_resources:` `["folders/<foo_folder1_id>", "projects/<foo_project1_id"]`
```
Comment (blueandgold) [inline _docs/_latest/configure/general/non-org-root.md]: nit: we should not use folder as example here

File: _docs/_latest/configure/general/non-org-root.md (hunk 1 above)
Comment (red2k18) [reply _docs/_latest/configure/general/non-org-root.md]: Done

File: _docs/_latest/configure/general/non-org-root.md (hunk 2 above)
Comment (red2k18) [reply _docs/_latest/configure/general/non-org-root.md]: Updated

**Pull request merged**